huggingface/BART.md

```diff
@@ -1,10 +1,16 @@
 # Run Instruction
 1. Follow steps in [README.md](README.md)
-2. Training script in [2.2 Run this recipe](README.md#2.2-Run-this-recipe) for BART is
+2. Launch script in [2.2 Run this recipe](README.md#2.2-Run-this-recipe) for BART
+If running on AzureML,
 ```bash
 cd huggingface/azureml
 python hf-ort.py --gpu_cluster_name <gpu_cluster_name> --hf_model bart-large --run_config ort
 ```
+If running locally, 
+```bash
+cd huggingface/azureml
+python hf-ort.py --hf_model bart-large --run_config ort --process_count <process_count> --local_run True
```

Comment:
Can we change the --local_run option to do a local run if --local_run is specified. 'True' shouldn't be needed.

Reply:
Changed